generate/src/contract/common.rs

```diff
@@ -86,13 +86,13 @@ pub(crate) fn expand(cx: &Context) -> TokenStream {
 
             /// Returns a reference to the raw runtime instance used by this
             /// contract.
-            pub fn raw_instance(&self) -> &self::ethcontract::DynInstance {
+            pub fn raw_instance(&self) -> &self::ethcontract::private::DynInstance {
```

Comment:
Is it correct that a public method returns a type that we say is private?

Reply:
Good point, I'll change it.